src/components/UserUpdate/UserUpdate.tsx

```diff
@@ -20,6 +23,10 @@ const UserUpdate: React.FC<UserUpdateProps> = ({ id }): JSX.Element => {
     firstName: '',
     lastName: '',
     email: '',
+    password: '',
+    applangcode: '',
+    selectedOption: '',
```

Comment:
@kb-0311 
These fields were removed previously. There is a separate component for password update.

Also, the applangcode and selectedOption are redundant. Since, they are stored while signing up.

Due to the changes in this file, the update user page is not working for now.

Reply:
@anshgoyalevil I was not aware of this, maybe the changes happened because of git conflicts resolution.

Reply:
@kb-0311 Should we proceed with a new issue, or a new PR linking this issue?

Reply:
@anshgoyalevil Yeah sure you can create the issue and fix it